tests/utils/customblocks/ReactImage.tsx

```diff
@@ -30,15 +30,17 @@ export const ReactImage = createReactBlockSpec({
           alt={"Image"}
           contentEditable={false}
         />
-        <InlineContent />
+        <InlineContent style={{ flexGrow: 1 }} />
```

Comment:
why was this added?

Reply:
It's just to make sure that the regular `Image` and `ReactImage` have identical styling, makes it easier to compare the serialized HTML to check that it's the same for vanilla and React.